src/System.Threading.Tasks.Channels/System/Threading/Tasks/Channels/ReadableChannel.cs

```diff
@@ -192,52 +223,5 @@ public void Dispose()
                 }
             }
         }
-
-        /// <summary>Gets an async enumerator of the data in this channel.</summary>
-        /// <param name="cancellationToken">The cancellation token to use to cancel the asynchronous enumeration.</param>
-        /// <returns>The async enumerator.</returns>
-        public virtual IAsyncEnumerator<T> GetAsyncEnumerator(CancellationToken cancellationToken = default) =>
```

Comment:
@davidfowl @anurse We use this for streaming in SignalR. Looks easy enough to make into an extension method for our internal use until a public API is added back.

@stephentoub Is there an ETA for this support, like 2.1?

Reply:
I removed it because I don't want it to conflict with the real `IAsyncEnumerable<T>/IAsyncEnumerator<T>` we intend to deliver along with C# 8.  I do not expect this to be in 2.1.
https://github.com/dotnet/csharplang/blob/master/proposals/async-streams.md

Reply:
Ok, I think we'll probably just copy this extension method into our code base as internal code (and plan to replace it with the real thing when that comes out).

Reply:
Sounds good.  Thanks.

Reply:
ZOMG 😄